src/api/endpoints/activity_types/activity_types.py

```diff
@@ -97,18 +97,26 @@ def put(self, act_types_id=None):
                 message="Resource not found."
             ), 404)
 
-        if payload.get("name"):
-            target_activity_type.name = payload.get("name")
-        if payload.get("description"):
-            target_activity_type.description = payload.get("description")
-        if payload.get("value"):
-            target_activity_type.value = payload.get("value")
-        if payload.get("supports_multiple_participants"):
+        result, errors = new_activity_type_schema.load(payload, partial=True)
+
+        if errors:
+            status_code = new_activity_type_schema.context.get(
+                'status_code')
+            validation_status_code = status_code or 400
+            return response_builder(errors, validation_status_code)
+
```

Comment:
Looping over the result dict would improve this.`<not critical>`
```python
    for key in result:
        setattr(target_activity_type,  key, result[key])
```

Reply:
Good suggestion. Much cleaner and concise. This would also work just as good.
```python
    for key, value in result.items():
        setattr(target_activity_type,  key, value)
```